cmd/kindnetd/main.go

```diff
@@ -94,29 +62,28 @@ func main() {
 		))
 	}
 
-	// used to store the routes
-	kindnetConfig := &KindnetConfig{
-		PodCIDRs: make(map[string]string),
-	}
-
 	// used to track if the cni config inputs changed and write the config
 	cniConfigWriter := &CNIConfigWriter{
 		path:     CNIConfigPath,
 		template: os.Getenv("CNI_CONFIG_TEMPLATE"),
 	}
+
+	// enforce ip masquerade rules
 	ipv6 := false
+	// TODO: dual stack...?
 	if strings.Contains(hostIP, ":") {
 		ipv6 = true
 	}
-	// used to create routes and iptables rules
-	kindnetd := NewKindnet(cniConfigWriter, ipv6, kindnetConfig)
+	masqAgent, _ := NewIPMasqAgent(ipv6, []string{os.Getenv("POD_SUBNET")})
+	go masqAgent.SyncRulesForever(time.Second * 60)
```

Comment:
what happens if there is an error with the masqAgent?

Reply:
right now it won't handle it. we could panic or log an error.

I think in this case we should panic just until a (near) future refactor adds proper logging

Reply:
updated to panic for now with a TODO to log pervasively.

xref: #109 I think it's about time to consider merging with the ecosystem on that front...

Reply:
thanks for catching this.